code/modules/reagents/chemistry/reagents.dm

```diff
@@ -149,6 +162,16 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 /datum/reagent/proc/on_mob_end_metabolize(mob/living/L)
 	return
 
+/// Called when a reagent is inside of a mob when they are dead
+/datum/reagent/proc/on_mob_dead(mob/living/carbon/C)
+	if(!chemical_flags & REAGENT_DEAD_PROCESS)
```

Comment:
```suggestion
	if(!(chemical_flags & REAGENT_DEAD_PROCESS))
```
`!` takes priority over `&`, so you want parentheses here.

Reply:
I see, thanks for picking this out. I'll remember this.